index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+    include_once 'database.class.php';
+
+    /* parse config data */
+    $config = parse_ini_file('config.ini');
+
+    $db = new Database($config['driver'], $config['host'], $config['port'], $config['username'], $config['password'], $config['dbname'], $config['unix_socket'], $config['charset']);
```

Comment:
@lumakos What If  I would like to connect at a MSSQL server and select data from a table, and paginate through the result set? Would this implementation be able to support this? Because it seems that is pretty much compatible with MySQL database

Reply:
I think this part of code should be more abstact, I should change it and build an abstract function about it. Generally , i tried to build an abstract databate manager but i did not give enough attention it this line before i commited it.